src/content/1.7/basics/keeping_up-to-date/migration.md

```diff
@@ -0,0 +1,678 @@
+---
+title: Migration
+weight: 20
+---
+
+# How to: Migrate
+
+## Manual migration - Process details
+
+We will use the ETL (Extract - Transform - Load) process to complete a migration, a concept usually applied to data warehouses.
+
+The database structure may change with each new release of PrestaShop, but major versions may also imply features being replaced, preventing a shop to be smoothly upgraded in some cases. For these shops, a migration is a solution to be able to keep up with new releases.
+
+We will cover the 3 main steps in this chapter:
+
+* Extract data to be transfered
+* Add / modify / delete some values if needed
+* Import data on the new shop
+
+### Exporting data from the source
+
+In difference with an upgrade, we won’t modify the database directly. We need to export the data first. We use CSV files here, the most compatible and generic format.
+
+Exporting data can be done
+
+* Via the PrestaShop administration panel,
+* With your MySQL client.
+
+Depending on the way you export your data, you won’t get the same amount of data in your exports.
+
+#### Light - Via the administration panel
+
+Exporting data is quite straightforward. On each controller dedicated to the data you want to export, you will find action buttons at the top right of the page:
+
+{{< figure src="/images/1.7/upgrade-migration/migration-export-bo-zoom.png" title="Export button in action bar" >}}
+
+For instance for your products on PrestaShop 1.7:
+{{< figure src="/images/1.7/upgrade-migration/migration-export-bo.png" title="Action bar localization" >}}
+
+Clicking on the “Export” button will make your browser download a CSV file, which contains the complete product list with the columns displayed on the page.
+
+With the same process, you can export the following data:
+* (Brand) Addresses
```

Comment:
You need to add a blank line before your first item, or it won't render as a list.